packages/react-components/react-list-preview/stories/List/ListActiveElement.stories.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Button,
+  makeStyles,
+  Persona,
+  shorthands,
+  mergeClasses,
+  Text,
+  tokens,
+  SelectionItemId,
+} from '@fluentui/react-components';
+import { Mic16Regular } from '@fluentui/react-icons';
+import { List, ListItem } from '@fluentui/react-list-preview';
+
+import * as React from 'react';
+const names = [
+  'Melda Bevel',
+  'Demetra Manwaring',
+  'Eusebia Stufflebeam',
+  'Israel Rabin',
+  'Bart Merrill',
+  'Sonya Farner',
+  'Kristan Cable',
+];
+
+type Item = {
+  name: string;
+  id: string;
+  avatar: string;
+};
+
+const items: Item[] = names.map(name => ({
+  name,
+  id: name,
+  avatar:
+    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
+}));
+
+const useStyles = makeStyles({
+  wrapper: {
+    display: 'grid',
+    gridTemplateColumns: '240px 1fr',
+    columnGap: '16px',
+  },
+  button: {
+    alignSelf: 'center',
+  },
+  item: {
+    cursor: 'pointer',
+    ...shorthands.padding('2px', '6px'),
+    justifyContent: 'space-between',
+  },
+  itemSelected: {
+    backgroundColor: tokens.colorSubtleBackgroundSelected,
+  },
+});
+
+// Memoizing the ListItem like this allows the unaffected ListItem not to be re-rendered when the selection changes.
+const MyListItem = React.memo(
+  ({ name, avatar, ...rest }: React.ComponentProps<typeof ListItem> & { name: string; avatar: string }) => {
+    const styles = useStyles();
+    return (
+      <ListItem key={name} value={name} aria-label={name} {...rest} checkmark={null}>
```

Comment:
Has @kolaps33  gone through these examples? I'm afraid that this kind of scenario should actually be with `role="tab"`?

Reply:
that makes sense, I'll update the story